### DISCUSSION TITLE
Missing extension.json | Kotlin

### DISCUSSION THREAD

**Discussion by aquabtww** [Minestom/Minestom#1819] [Help] [answered]
Hello, i've been trying to create an extension, and i keep getting this error no matter what i do, even tho the extension.json is there.
![image](https://user-images.githubusercontent.com/54704830/230185647-f969b681-f97b-4a41-981b-cf8b83fe2ef5.png)

**Comment by ZakShearman** [comment, marked as answer]
Your file is called extension**s**.json, not extension.json
